### PR TITLE
feat: Automate course Videos dashboard screen

### PR DESCRIPTION
**Pull request by bilalawan321**
[LEARNER-7791](https://openedx.atlassian.net/browse/LEARNER-7791)

**Description**

Verify that from My Courses list tapping any Course will load Course
Dashboard screen/contents(of this specific course) in its tab
Verify on tapping "Discussion" tab will load Discussions screen
Verify on tapping "Dates" tab will load Dates screen
Verify on tapping "Resources" tab will load Resources list
Verify on tapping "Videos" tab will load Videos screen
Verify that the Course Videos Dashboard screen will show the following contents,
Header contents, Back icon, Specific "<course name>" as Title, Share icon,
Footer Content,
Course, Videos, Discussion, Dates, Resources
Verify that the user should be able to view these contents:
videos download header,
Videos download switch

### CODE REVIEW

File: tests/ios/pages/ios_my_courses_list.py [hunk 1]
```diff
@@ -32,13 +32,13 @@ def get_my_courses_list_row(self):
...
 
-        courses_row = self.global_contents.get_all_views_on_ios_screen(
+        courses_row = self.global_contents.get_all_elements_by_id(
             self.driver,
             ios_elements.my_courses_list_course_row
         )
 
-        if courses_row:
-            return courses_row[0]
+        if courses_row[1]:
```
Comment (jawad-khan) [inline tests/ios/pages/ios_my_courses_list.py:40]: Prefer one liners like:

return courses_row[1] or courses_row
OR 

return courses_row[1] if courses_row[1] else courses_row

File: tests/ios/tests/test_ios_videos_dashboard.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            Verify that user should be able to view these contents:
+                videos download header,
+                Videos download switch
+        """
+
+        global_contents = Globals(setup_logging)
+        ios_course_dashboard_page = IosCourseDashboard(set_capabilities, setup_logging)
+        ios_videos_dashboard_page = IosVideosDashboard(set_capabilities, setup_logging)
+
+        assert ios_course_dashboard_page.get_courses_tab().text == global_contents.is_selected
```
Comment (jawad-khan) [inline tests/ios/tests/test_ios_videos_dashboard.py:78]: can we break this function. Too much assertion happenings in one place.

Comment (bilalawan321) [reply tests/ios/tests/test_ios_videos_dashboard.py:71]: fixed

File: tests/ios/pages/ios_my_courses_list.py (hunk 1 above)
Comment (bilalawan321) [reply tests/ios/pages/ios_my_courses_list.py]: fixed

**Review by jawad-khan — APPROVED** [review]

**Pull request merged**